test/sample-test/sample_test_launcher.py

```diff
@@ -129,7 +129,9 @@ def _compile(self):
           self._run_pipeline = raw_args['run_pipeline']
 
       if self._run_pipeline:
-        nb_params['experiment_name'] = self._test_name + '-test'
+        if not raw_args['notebook_params']['experiment_name']:
+          raise ValueError('Experiment name is required if pipeline run is expected.')
+        nb_params['experiment_name'] = raw_args['notebook_params']['experiment_name']
```

Comment:
The fact that the experiment_name value in the config yaml is not used(but generated by the test infra) should be noted in the schema yaml file, similar to the output value not being used. WDYT?